frontend/src/hooks/use-mutate.js

```diff
@@ -48,9 +48,31 @@ export const useMutate = () => {
       }
     })
 
+    const addTag = useMutation((input) => gqlHelper(ADD_NEW_TAG, input),
+    {
+      onSuccess: (response) => {
+        checkResponse(response)
```

Comment:
`checkResponse` (for demo purposes) is currently only invalidating the products query. Since this is not a product that we're updating, we want to invalidate all tags here (or whatever the react-query name you guys set was)